lib/src/reactive_ble.dart

```diff
@@ -176,8 +176,6 @@ class FlutterReactiveBle {
     QualifiedCharacteristic characteristic, {
     @required List<int> value,
   }) async {
-    await initialize();
-
     await initialize();
```

Comment:
Which doesn't _really_ hurt, but not needed either.

Reply:
ow wow good catch not super efficient indeed